web-components/collapse/collapse.svelte

```diff
@@ -0,0 +1,131 @@
+<svelte:options tag="leo-collapse" />
+
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte'
+
+  export let title = ''
+  export let isOpen: boolean | undefined
+  $: isOpenInternal = isOpen ?? false
```

Comment:
Why do we need two variables here? I believe this could be changed to

```suggestion
  export let isOpen: boolean | undefined = false;
```

Reply:
Ah yeah, Pete asked this as well. It's so we can have a controlled and uncontrolled state for the collapse. There might be a better way of doing this in Svelte but basically, I wanted it to work normally without specifying `isOpen` on the component (it should toggle itself) and also in the React way, where you set `isOpen` and it doesn't toggle when you click it, it just emits an event.

Reply:
I think I understand. In your example of a more idiomatic solution, I don't think `bind:` is necessary?

Reply:
Yeah, so basically, I want to support both of these cases:

```svelte

<!-- Parent doesn't care -->
<Parent>
	<Collapse>Stuff & Things</Collapse>
</Parent>

<!-- Parent is in charge -->
<script>
     let open = false;
</script>
<button on:click={() => open = false}>Close the thing!</button>
<Collapse isOpen={open} on:toggle={newOpen => open = newOpen}>Maybe hidden?</Collapse>
```

<!-- Maybe a more sveltey way? I dunno. This doesn't work as is -->
<script>
    let open = false;
</script>
<button on:click={() => open = false}>Close the thing 2.0!</button>
<Collapse bind:isOpen={open}>Maybe hidden?</Collapse>